data/yam/agama/auto/agama_auto_tumbleweed.yaml

```diff
@@ -0,0 +1,3 @@
+---
+test_data:
+  product: openSUSE Tumbleweed
```

Comment:
you are confusing here the path, look at this example:
https://openqa.suse.de/tests/11164022#settings
path should under `test_data/yam/`
And you can already include the rest of the products in other files.